src/components/ui/Pool/index.tsx

```
@@ -63,18 +65,26 @@ export const Pool = () => {
     isTxConfirming,
     isConfirmed,
-    receipt,
+    receipt: txReceipt,
     claimFees,
   } = useClaimFees();
 
   useEffect(() => {
     if (isConfirmed) {
       openClaimSuccessModal();
+
+      const totalFee = (txReceipt!?.gasUsed * txReceipt!?.effectiveGasPrice);
```

Biome

[error] Forbidden extra non-null assertion.

Safe fix: Remove extra non-null assertion.


(lint/suspicious/noExtraNonNullAssertion)